File: EASY/ONBOARDING/php/onboarding.php
<?php
// todo: сделаем короче?
while (TRUE)
{
    fscanf(STDIN, "%s",
        $enemy1 // name of enemy 1
    );
    fscanf(STDIN, "%d",
        $dist1 // distance to enemy 1
    );
    fscanf(STDIN, "%s",
        $enemy2 // name of enemy 2
    );
    fscanf(STDIN, "%d",
        $dist2 // distance to enemy 2
    );

    if ($dist1 < $dist2) {
        echo ($enemy1."\n");
    } else {
        echo ($enemy2."\n");
    }
}
?>

File: EASY/ONBOARDING/f#/onboarding.fs
open System

while true do
    let enemy1 = Console.In.ReadLine() (* name of enemy 1 *)
    let dist1 = int(Console.In.ReadLine()) (* distance to enemy 1 *)
    let enemy2 = Console.In.ReadLine() (* name of enemy 2 *)
    let dist2 = int(Console.In.ReadLine()) (* distance to enemy 2 *)

    if dist1 < dist2 then printfn "%s" enemy1
    else printfn "%s" enemy2

()